tests/unit/store/alert.spec.js

```diff
@@ -0,0 +1,89 @@
+import store from '@/store'
```

Comment:
This will be a bit easier to read (and follow stylelint guidelines) with some spacing between your describe blocks